### PR TITLE
Update changelog for 4.43.0

### PR DESCRIPTION
**Pull request by babakpst**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,52 @@
 # Changelog for Tensile
 
-## (Unreleased) Tensile 4.42.0
+## (Unreleased) Tensile 4.43.0
+
+### Added
+
+- Nightly builds and groovy
```
Comment (bstefanuk) [inline CHANGELOG.md:7]:
```suggestion
- Nightly builds and with performance statistics
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,52 @@
...
+- Guard against inconsistent state
+- Cache asm capabilities for reuse
+- venv for Tensile create on Linux
+- Flag to keep build_tmp when running Tensile
+- Generalized profiling scripts
+- GFX1151 support
+- Single-threaded operations support, mainly for profiling
```
Comment (bstefanuk) [inline CHANGELOG.md:14]:
```suggestion
- Single-threaded operation support in TensileCreateLibrary
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,52 @@
 # Changelog for Tensile
 
-## (Unreleased) Tensile 4.42.0
+## (Unreleased) Tensile 4.43.0
+
+### Added
+
+- Nightly builds and groovy
+- Guard against inconsistent state
```
Comment (bstefanuk) [inline CHANGELOG.md:8]: I think this can be removed

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,52 @@
...
+- Generalized profiling scripts
+- GFX1151 support
+- Single-threaded operations support, mainly for profiling
+- Logic to remove temporary build artifacts
+
+### Changed
+
+- Updated Tensile documents (API reference, README.md, and comments)
+- Disabled asm-cache for tests
+- Used hipcc.bat as a compiler on windows instead of the Perl script
```
Comment (bstefanuk) [inline CHANGELOG.md:21]:
```suggestion
- Used hipcc.bat as a compiler on Windows instead of the Perl script
```

**Review by bstefanuk — APPROVED** [review]

**Review by ellosel — APPROVED** [review]

**Pull request merged**